### PR TITLE
schema/: merge Project and Location extensions

### LINKED ISSUE #1484: Merge the Project and Location extension to encourage implementers to publish infrastructure project identifiers
Currently, the field for disclosing an infrastructure project identifier is part of the [Project](https://extensions.open-contracting.org/en/extensions/project/master/) extension, which is referenced from the OCDS docs on the page about [linking to other standards](https://standard.open-contracting.org/latest/en/guidance/map/linked_standards/?highlight=infrastructure#open-contracting-for-infrastructure-data-standards).

To increase the chances of OCDS publishers including infrastructure project ids in their OCDS data, even when they aren't implementing OC4IDS, we could merge the extension (or at least the `planning/project/id field`) into the main schema. That way, it would be visible to OCDS implementers as part of the mapping process.

We could also document a worked example/guidance page on publishing infrastructure project identifiers, which could reuse some of the content from the project identifiers guidance in the OC4IDS documentation.

### PR DESCRIPTION
**Pull request by odscjen**
closes #1484 

was unable to run manage.py, got the following error

> Exception: project is multilingual at planning/budget/project, but not at planning/project

which I think is because `planning/budget/project` is a string field and `planning/project` is an object.

### CODE REVIEW

**Comment by odscjen** [comment]
@jpmckinney does an exception need to be added to `manage.py` to allow `planning.project` to be nonmultilingual?

**Comment by jpmckinney** [comment]
> @jpmckinney does an exception need to be added to manage.py to allow planning.project to be nonmultilingual?

I'll make the necessary changes once the PR is ready for review.

**Comment by odscjen** [comment]
Thanks @jpmckinney this is ready for review now

**Comment by odscjen** [comment]
spell check false positive, picking up on "sme"

**Review by jpmckinney — APPROVED** [review]
I made several changes. Please review. Mainly:

- Put information in the schema (which we know people read) rather than the docs (which we know people read less frequently).
- Tell people what to do, rather than tell them what not to do. This meant making budget.projectID clearer.
- Remove sentences that didn't have a lot of content.
- Don't repeat GeoJSON norms.

File: schema/release-schema.json [hunk 1]
```diff
@@ -2736,6 +2883,82 @@
...
+          "description": "An externally provided identifier for the project. This can be a URI, an [Open Contracting for Infrastructure Data Standards Toolkit (OC4IDS)](https://standard.open-contracting.org/infrastructure/latest/en/guidance/identifiers/) project identifier, [OCDS for PPPs](https://standard.open-contracting.org/profiles/ppp/latest/en/framework/#i-basic-project-information) project identifier, an identifier from a projects register, or an identifier based on the canonical version of the project's name. Project identifiers should be unique within the scope of the publication.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "minLength": 1
+        },
+        "title": {
+          "title": "Title",
+          "description": "The name of the project. If a projects register is available, this should be the same as the project's name in that register.",
```
Comment (odscjen) [inline schema/release-schema.json]:
```suggestion
          "description": "The name of the project. If a project's register is available, this should be the same as the project's name in that register.",
```

**Comment by odscjen** [comment]
1 small typo that I've just gone ahead and corrected. Otherwise all changes look good

**Comment by jpmckinney** [comment]
I'm not sure that was a typo? We typically talk about a projects register (register of projects) not an (individual) "project's register".

**Comment by odscjen** [comment]
oops, sorry I'd misread the sentence! I'll do a quick PR to undo that change rather than trying to rollback seeing as the branch has already been deleted

**Comment by odscjen** [comment]
ah I see you've fixed that elsewhere, thank you!

**Pull request merged**